docs/release-notes/8.7.asciidoc

```diff
@@ -0,0 +1,83 @@
+[[release-notes-header-8.7.0]]
+== 8.7
+
+[discrete]
+[[release-notes-8.7.0]]
+=== 8.7.0
+
+[discrete]
+[[known-issue-8.7.0]]
+==== Known issues
+* After alerts are generated for the first time, you may have to refresh your browser before your alert data appears on pages that use data views (for example, Timeline). Navigating between pages will not work (https://github.com/elastic/security-docs/issues/3046[#3046]).
+
+[discrete]
+[[breaking-changes-8.7.0]]
+==== Breaking changes
+
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: https://github.com/elastic/kibana/pull/
+There are no breaking changes in 8.7.0.
+// end::breaking-changes[]
+
+
+[discrete]
+[[deprecations-8.7.0]]
+==== Deprecations
+There are no deprecations in 8.7.0.
+
+
+[discrete]
+[[features-8.7.0]]
+==== Features
+* Creates a new dashboard, Data Quality, which highlights any issues in your ECS field mappings ({pull}150063[#150063]).
+* Introduces a new event type (`Credential access`) to represent credential dumping attempts on Windows by tools like Mimikatz or fgdump.
+* Creates a Torq connector that can trigger Torq workflows ({pull}149405[#149405]).
+* Adds more key performance indicator charts to the Alerts page ({pull}150242[#150242], {pull}149173[#149173], and {pull}146938[#146938]).
+* Adds filters to the Alerts page ({pull}146989[#146989]).
```

Comment:
The Alerts page filters are behind a feature flag for 8.7, so the feature has essentially been pushed back to 8.8.